docs/sql/kill.txt

```diff
@@ -0,0 +1,40 @@
+.. highlight:: psql
+.. _kill_job:
+
+=========
+Kill Jobs
+=========
+
+Each executed sql statement results in a corresponding job.
+Jobs that are currently executed are logged in the system table ``sys.jobs`` 
+(see :ref:`jobs_operations_logs`).
+
+To obtain the `UUID`_ of a job you need to enable stats 
+(see :ref:`conf_collecting_stats`). Jobs are all logged into ``sys.jobs_log``. 
```

Comment:
no need to mention where jobs are logged to, because you already have the reference.
saying how to prevent logging when stats are enabled is enough ;)